theme_graphene/views/customizations.xml

```diff
@@ -135,29 +135,21 @@
         <attribute name="class" add="pb64" remove="pb48" separator=" "/>
     </xpath>
     <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//h4" position="replace" mode="inner">
-        Tony Fred, <small><b>CEO</b></small>
-    </xpath>
-    <xpath expr="//div[hasclass('s_nb_column_fixed')]/div//p" position="replace">
-        <small>Founder and chief visionary, Tony is the driving force behind the company. He loves to keep his hands full by participating in the development of the software, marketing, and customer experience strategies.</small>
+        Tony Fred, <small><b>CTO</b></small>
```

Comment:
- CTO => **CEO** ;)

- Also, `<small>` and `<b>` seem to have no effect ...

![image](https://github.com/user-attachments/assets/6334aa78-256d-4353-93da-f7a23d983ab2)

- Why not use a subtitle as in the default snippet?

Reply:
Nice catch. I removed the customisations for this one as they are not really needed. Thanks !